simplesource-command-serialization/src/main/java/io/simplesource/kafka/serialization/avro/AvroSerdes.java

```diff
@@ -207,7 +198,7 @@ private static Schema aggregateUpdateResultSchema(final Schema aggregateSchema)
         private static GenericRecord fromReason(final Schema schema, final CommandError commandError) {
             return new GenericRecordBuilder(schema)
                     .set(ERROR_MESSAGE, commandError.getMessage())
-                    .set(ERROR_CODE, commandError.getReason().name())
+                    .set(ERROR, Base64.getEncoder().encodeToString(SerializationUtils.serialize(commandError)))
```

Comment:
With Avro, we can maybe be more efficient in terms of space use by using `Base64.getEncoder().encode` in place of `Base64.getEncoder().encodeToString` and declare the field as a `byteType` here https://github.com/simplesourcing/simplesource/pull/38/files#diff-717500181ed887fe65e9940056d96a5cR236

but the problem is still the same in terms of time use and doesn't help for the JSON serialization.

Reply:
I tested. I can't make the previous proposition work. I don't know why.

Reply:
That would definitely be better for Avro serialization.

Making the field a `bytesType` should work. You may need to wrap the `byte[]` as a `ByteBuffer`
`ByteBuffer.wrap(SerializationUtils.serialize(commandError))` or something like that.

For Json, a Base64 encoded string would be acceptable.